docs/09.EXT-REFERENCE-ARG.md

```diff
@@ -206,6 +231,39 @@ jerryx_arg_transform_args (const jerry_value_t *js_arg_p,
 - [jerryx_arg_transform_this_and_args](#jerryx_arg_transform_this_and_args)
 
 
+## jerryx_arg_transform_object_properties
+
+**Summary**
+
+Validate the properties of a target JS object and assign them to the native arguments.
```

Comment:
I'd remove the word "target", I think.
When I read this, it makes me think we're setting something on the "target" / we're "targeting" the JS object.